src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -837,7 +838,7 @@ private static IntermediateNode GetBooleanBinaryOp(IRTranslatorContext context,
                     }
                     else
                     {
-                        throw new NotSupportedException();
+                        return new BooleanLiteralNode(context.GetIRContext(node), false);
```

Comment:
This should probably be true for neq, I would guess.

Reply:
![image](https://user-images.githubusercontent.com/69215460/164785974-28448770-d205-4e00-b77a-3190229c0a80.png)

Reply:
Fixed